xlators/mgmt/glusterd/src/glusterd-store.c

```diff
@@ -3014,6 +3014,29 @@ glusterd_store_retrieve_node_state(glusterd_volinfo_t *volinfo)
     return ret;
 }
 
+/* change slave$i to secondary$i, where i = 1,2,3,...*/
+static void
+glusterd_chk_update_geo_rep_key_name(char **key)
```

Comment:
Added this function to change the key slave1 to secondary1 in `/var/lib/glusterd/vols/$volname/info` during upgrade from old to new version, when glusterd is restarted.